### PR TITLE
feat: implement TLS support

### LINKED ISSUE #32: Add TLS support

### PR DESCRIPTION
**Pull request by aholovko**
Closes #32

Signed-off-by: Andriy Holovko <andriy.holovko@gmail.com>

### CODE REVIEW

File: test/bdd/fixtures/kms-rest/docker-compose.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ services:
     image: ${KMS_REST_IMAGE}:latest
     environment:
       - KMS_REST_HOST_URL=0.0.0.0:8070
-      - KMS_REST_TLS_CACERTS=/etc/tls/ec-cacert.pem
```
Comment (DRK3) [inline test/bdd/fixtures/kms-rest/docker-compose.yml:15]: Were these two env vars here by mistake before? Doesn't look like those args existed

File: cmd/kms-rest/startcmd/start_test.go [hunk 2]
```diff
@@ -29,13 +29,13 @@ const (
```
Comment (DRK3) [inline cmd/kms-rest/startcmd/start_test.go:29]: I think the GetTLS method is what's keeping your code coverage low... Take a look at the unit tests in the EDV project - you should be able to do the same thing here to get your code coverage up

File: test/bdd/fixtures/kms-rest/docker-compose.yml (hunk 1 above)
Comment (aholovko) [reply test/bdd/fixtures/kms-rest/docker-compose.yml:15]: They are leftovers from my previous refactoring and initially were blindly added during bootstrapping the project.

File: cmd/kms-rest/startcmd/start_test.go (hunk 2 above)
Comment (aholovko) [reply cmd/kms-rest/startcmd/start_test.go:30]: Thank you Derek for that suggestion. For some reason, I didn't realize that just adding the flag with an empty value can trigger needed failure behavior for that flag. So your example code was very helpful here.

**Pull request merged**